From 1e8da00fbdb57afbe201bd6e803b11f0227b4d1a Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 28 Aug 2020 09:00:20 +0800
Subject: [PATCH] ARM: dts: rv1126: Modify npu opp table

Add more test case for Vmin test, such as mobilenet_v1, ssd mobilenet v2,
style transfer.

Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
Change-Id: Ibe5cdf6c27a28b1153e9d42e16f848b3699fc121
---
 arch/arm/boot/dts/rv1126.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 53627fcea507..74591090bc5e 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -2328,19 +2328,19 @@
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-			opp-microvolt = <725000 725000 1000000>;
+			opp-microvolt = <750000 750000 1000000>;
 		};
 		opp-700000000 {
 			opp-hz = /bits/ 64 <700000000>;
-			opp-microvolt = <775000 775000 1000000>;
+			opp-microvolt = <800000 800000 1000000>;
 		};
 		opp-800000000 {
 			opp-hz = /bits/ 64 <800000000>;
-			opp-microvolt = <825000 825000 1000000>;
+			opp-microvolt = <850000 850000 1000000>;
 		};
 		opp-934000000 {
 			opp-hz = /bits/ 64 <934000000>;
-			opp-microvolt = <900000 900000 1000000>;
+			opp-microvolt = <950000 950000 1000000>;
 			status = "disabled";
 		};
 	};
-- 
2.35.3

